### ISSUE TITLE
Main Menu issue - It show in all the views

### ISSUE THREAD

**Issue by nachommm** [dhanasingh/redmine_wktime#130]
Hi there once I have installed the plugin I got the surprice that main-menu it show in all the views, let me show you an example, where main menu it show in login page.

![image](https://user-images.githubusercontent.com/6294983/33371622-24ed1e5a-d4da-11e7-8c9c-772715b49b65.png)

I have installed this version of redmine an plugins:

Environment:
  Redmine version                3.4.2.stable
  Ruby version                   2.1.4-p265 (2014-10-27) [x86_64-linux]
  Rails version                  4.2.8
  Environment                    production
  Database adapter               Mysql2
SCM:
  Subversion                     1.9.3
  Git                            2.7.4
  Filesystem                     
Redmine plugins:
  additionals                    2.0.5
  issue_charts                   1.3.0
  progressive_projects_list      3.0.1
  redmine_checklists             3.1.9
  redmine_ckeditor               1.1.5
  redmine_contacts               4.1.1
  redmine_dmsf                   1.6.0
  redmine_favorite_projects      2.0.3
  redmine_local_avatars          1.0.3
  redmine_wktime                 3.1

**Comment by nachommm** [comment]
Well, I find the issue an got the changes, now is working...

I've changed the render_main_menu on init.rb

```
> def render_main_menu(project)
>   if menu_name = controller.current_menu(project)
>        render_menu(menu_name(project), project) 
>    end
> end
```

**Comment by karthickmadhiazhagan** [comment]
Thanks for reporting. We have added this fix in v3.3. Please take a look.
